recipes/zarray/build.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
-￼
```

Comment:
For some reason this odd character was your file.

Reply:
Thanks a lot!